Integrations/SymantecMSS/SymantecMSS.py

```diff
@@ -74,15 +77,20 @@ def load_severities():
 ''' HELPER FUNCTIONS '''
 
 
+def strip_unwanted_chars(s):
+    return re.sub('&.+[0-9]+.?;', '', s)
```

Comment:
as you don't have any match group here this will do nothing

Reply:
@yaakovi What do you mean? This takes all &4954; words and removes them from s. This is exactly what fixes the bug